### PR TITLE
Release 1.6.1

### PR DESCRIPTION
**Pull request by kosonenj**
### Summary and Scope

Description: 

Canu 1.6.1 release

PR checklist (you may replace this section):
- [x] I have run `nox` locally and all tests, linting, and code coverage pass.
- [x] I have added new tests to cover the new code
- [x] My code follows the style guidelines of this project
- [x] If adding a new file, I have updated pyinstaller.py
- [x] I have updated the appropriate Changelog entries in readme.md
- [x] I have incremented the version in the readme.md
- [x] I have incremented the version in `canu/.version` and `.version`

## [1.6.1]

- Disable load balacing configuration for Dell CDU/Leaf.

## [1.6.0]

- Add `canu report network version` feature.
- Fix Errors in the output of `canu test`

## [1.5.14]

- Add route-map and prefixes to allow connection to UAI's from CAN network.

## [1.5.13]

- Fix Dell4148 template to include correct port count 

## [1.5.12]

- Add netutils pyinstaller hook file.

## [1.5.11]

- Create unique VSX system macs for each VSX cluster.
- Fixed Mellanox Customer ACL.
- Add VLAN 7 to Dellanox UAN for 1.0

## [1.5.10]

- Fix canu paddle-file.json schema

## [1.5.9]

- Change Rosetta/Columbia switch naming to be sw-hsn-<rack>-<###> (as with PDU and CMM/CEC).
- Change switch port/interface descriptions to `dst:slot:port==>src` to avoid truncation.
- Change gateway nodes to 4 port 1G OCP card definitions.
- Add dvs and ssn nodes as 4 port 1G OCP card definitions.
- Change large memory node common name from `lm` to `lmem`.
- Beta release of `--reorder` for switch/network config generation where custom-config is not used.

## [1.5.8]

- Added shellcheck GitHub action
- Bump ipython to 7.16.3 to remediate CVE
- Clean up Jenkins build

## [1.5.7]

- Add ACL to block CHN <> traffic for CSM 1.2
- Add Route-Map to CMN BGP peers to restrict routes to only CMN IPs

### Testing

Tested on: Nox

### CODE REVIEW

File: canu/.version [hunk 1]
```diff
@@ -1 +1,2 @@
-1.5.12
+1.6.1~develop
```
Comment (lukebates123) [inline canu/.version:1]: remove ~develop

Comment (lukebates123) [inline canu/.version:1]: remove ~develop

File: readme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# 🛶 CANU v1.5.12
+# 🛶 CANU v1.6.1-develop
```
Comment (lukebates123) [inline readme.md:1]: remove -develop

Comment (lukebates123) [inline readme.md:1]: remove -develop

**Review by lukebates123 — CHANGES_REQUESTED** [review]

**Review by lukebates123 — APPROVED** [review]

**Review by trad511 — APPROVED** [review]

**Pull request merged**